NEWS.md

```diff
@@ -1326,7 +1345,8 @@ Compile with wlroots 0.12.0 and wayland-server >=1.16
   ShowMenu
 
 [Keep a Changelog]: https://keepachangelog.com/en/1.0.0/
-[unreleased]: https://github.com/labwc/labwc/compare/0.7.1...HEAD
+[unreleased]: https://github.com/labwc/labwc/compare/0.7.2...HEAD
+[0.7.2]: https://github.com/labwc/labwc/compare/0.7.0...0.7.2
```

Comment:
this should likely compare to 0.7.1 rather than 0.7.0

Reply:
Thanks for catching. Will change tomorrow.